src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -250,18 +181,17 @@ private void init()
         }
     }
 
-    public static class VillagerProfession
+    public static class VillagerProfession extends IForgeRegistryEntry.Impl<VillagerProfession>
```

Comment:
Maybe, we may look into that when its cleaned up but for now its gunna sit in place.